src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -179,7 +180,7 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup Condition="'$(StaticICULinking)' == 'true' and '$(NativeLib)' != 'Static' and '$(InvariantGlobalization)' != 'true'">
       <NativeLibrary Include="$(IntermediateOutputPath)libs/System.Globalization.Native/build/libSystem.Globalization.Native.a" />
-      <DirectPInvoke Include="System.Globalization.Native" />
+      <DirectPInvoke Include="libSystem.Globalization.Native" />
```

Comment:
I think this change can stay - so that the lib prefix is omitted everywhere in our build scripts.